### PR TITLE
New package : serviio-2.1_1

### PR DESCRIPTION
**Pull request by pandom79**
This is a free dlna media server.

### CODE REVIEW

File: srcpkgs/serviio/template [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+maintainer="Domenico Panella <pandom79@gmail.com>"
+license="custom"
+homepage="http://www.serviio.org/"
+distfiles="http://download.serviio.org/releases/${pkgname}-${version}-linux.tar.gz"
+checksum=51748160b38e31ba54db1b11381d3d9d705852ed7757ad111e881327a16fb757
+
+do_install() {
+
+    #Define variable for Serviio home directory
+    SERVIIO_HOME=/usr/share/java/$pkgname
```
Comment (ericonr) [inline srcpkgs/serviio/template:17]: This path shouldn't have a leading slash., if it's going to be used for `vmkdir` statements.

File: srcpkgs/serviio/template [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    vmkdir ${SERVIIO_HOME}
+
+    #Bin folder
+    vmkdir /usr/bin
```
Comment (ericonr) [inline srcpkgs/serviio/template:21]: This one shouldn't either.

File: srcpkgs/serviio/template [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    vcopy bin ${SERVIIO_HOME}
+    ln -s ${SERVIIO_HOME}/bin/${pkgname}-console.sh ${PKGDESTDIR}/usr/bin/${pkgname}-console
```
Comment (ericonr) [inline srcpkgs/serviio/template:23]: I think DESTDIR is the preferred var for this.

File: srcpkgs/serviio/template (hunk 2 above)
Comment (ericonr) [reply srcpkgs/serviio/template:21]: There are some other paths with the same issue.

**Comment by ericonr** [comment]
The commit message doesn't need to include the package revision, just the version. So "New package: serviio-2.1." should be fine. I don't think the body is necessary either, changed files can be viewed with `git log --name-status`.

**Comment by pandom79** [comment]
> The commit message doesn't need to include the package revision, just the version. So "New package: serviio-2.1." should be fine. I don't think the body is necessary either, changed files can be viewed with `git log --name-status`.

ok. thanks

**Comment by Chocimier** [comment]
>   c. The Software is intended for personal use only. Organizations and business 
      entities must purchase a business licence to operate the Software in a business 
      setup.   

Not redistributable by Void?

**Comment by pandom79** [comment]
I have updated my template how you suggest.

**Comment by pandom79** [comment]
I don't understand if it can be redistributable by void.
After few days, if you don't have a license ,  it pass automatically from business to free version. The licence string has been copy from another distro linux. What is the problem? Thanks

**Comment by abenson** [comment]
>    b. You may not sell, rent, loan or otherwise encumber the Software in whole or in part, standalone or as a part of integrated solutions, to any third party.

This is definitely not redistributable.

**Comment by pandom79** [comment]
ok. thanks

**Comment by ericonr** [comment]
@pandom79 the package can still be made "restricted", which means users have to build it themselves. If you're interested in re-opening the PR, it can be discussed.

**Comment by pandom79** [comment]
how to open this PR? I have opened another PR about signond crash that i solved on my laptop.

**Comment by pandom79** [comment]
> @pandom79 the package can still be made "restricted", which means users have to build it themselves. If you're interested in re-opening the PR, it can be discussed.

we can discuss [here](https://github.com/void-linux/void-packages/pull/23230)

**Comment by Chocimier** [comment]
> how to open this PR? I have opened another PR about signond crash

You need to make separate branch for every PR, containing only relevant changes.

**Comment by pandom79** [comment]
> > how to open this PR? I have opened another PR about signond crash
> 
> You need to make separate branch for every PR, containing only relevant changes.

After new brach created, i run 
```
git add <relevant changes>
git commit -S 
git push -f <fork/new-branch>
```
is correct?

**Pull request closed without merging**